tsconfig.json

```diff
@@ -5,5 +5,14 @@
     "resolveJsonModule": true,
     "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true
-  }
-}
+  },
```

Comment:
Changes only needed as you moved types. Leave types as is, or if critical to move, subit a separate PR moving types and updating build CI files.